### DISCUSSION TITLE
Autostart daemon doesnt work

### DISCUSSION THREAD

**Discussion by kf4ape** [Motion-Project/motion#1809] [Q&A]
I have tried to start with systemctl additions and also doing the /etc/default/motion item...still have to manually kick off the command...alternatively I could do cron...anyone else run into this?

**Comment by Mr-Dave** [comment]
`systemctl` does work.  Check status with `sudo systemctl status motion`  

`daemon` must be specified as `off` when using `systemctl`.  

/etc/default/motion is not used anymore.
